server/internal/handler/git.go

```
@@ -0,0 +1,1066 @@
+		return nil, err
+	}
+	body, err := os.ReadFile(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return []AgentSnapshot{}, nil
+		}
+		return nil, err
+	}
+	var items []AgentSnapshot
+	if err := json.Unmarshal(body, &items); err != nil {
+		return []AgentSnapshot{}, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 259) but it returns nil

(nilerr)